lib/teleterm/daemon/daemon.go

```diff
@@ -520,6 +524,13 @@ func (s *Service) SetGatewayTargetSubresourceName(ctx context.Context, gatewayUR
 		return nil, trace.Wrap(err)
 	}
 
+	if err := s.checkIfGatewayAlreadyExists(gateway.TargetURI(), CreateGatewayParams{
```

Comment:
There is a small bug in UI related to this feature. If I try to set the active port again, I'm getting an error. I think we should only try to make an API call on the UI side if the new port differs from the current one.

https://github.com/user-attachments/assets/7a48347a-df4d-473b-9f8f-b38658effe27

Reply:
Now that port buttons are gone I think I can get away with not changing this?

Reply:
Yeah, I think so.